C++/source/string/edit_distance.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+
+Edit Distance Algorithm :-
+--------------------------------
+Description: Find the minimum number of operations required to convert one string into another.
+
+Time Complexity :- O(m * n)
+Space Complexity :- O(m * n)
+                    Where m = length of string1 and n = string2
+
+*/
+
+
+
+#include<bits/stdc++.h>
+
+using namespace std;
+ 
+int min(int x, int y, int z) {
+   return min(min(x, y), z);
+}
```

Comment:
No need for this function. You can instead use `min({some_val, some_other_val, yet_another_val})` in your code.